ParallelBuildsMonitorWindowCommand.cs

```diff
@@ -154,7 +153,6 @@ void BuildEvents_OnBuildBegin(vsBuildScope Scope, vsBuildAction Action)
 
         void BuildEvents_OnBuildDone(vsBuildScope Scope, vsBuildAction Action)
         {
-            DataModel.CollectPerformanceData(true);
```

Comment:
Not needed.
Performance data will be collected independently from project begin/end. IMO that is even better, since they are some special points